install/app-chrome.sh

```diff
@@ -1,5 +1,7 @@
-cd /tmp
+#!/usr/bin/env bash
+
+cd ~/Downloads || exit
```

Comment:
Looks like when you resolved conflicts with https://github.com/basecamp/omakub/commit/ec7655e75e6a8806c49b16c624c56ea75ff1eb51, this one wasn't handled correctly: 

```suggestion
cd /tmp || exit
```

Reply:
Nice catch @rmacklin